tests/e2e/smoke.test.ts

```diff
@@ -0,0 +1,221 @@
+import { expect, test } from '@playwright/test'
+
+test.describe('Smoke Tests', () => {
+  test('app loads and displays welcome screen', async ({ page }) => {
+    await page.goto('/')
+
+    // Wait for the app to hydrate
+    await page.waitForLoadState('networkidle')
+
+    // Should display the welcome heading (for unauthenticated users)
+    const heading = page.locator('h1')
+    await expect(heading).toBeVisible({ timeout: 10000 })
+    await expect(heading).toContainText('Tinfoil Private Chat')
+  })
+
+  test('chat input is present and functional', async ({ page }) => {
+    await page.goto('/')
+    await page.waitForLoadState('networkidle')
+
+    // Chat input should be visible (on desktop)
+    const textarea = page.locator('textarea')
+
+    // On desktop, the textarea should be visible
+    const viewportSize = page.viewportSize()
+    if (viewportSize && viewportSize.width >= 768) {
+      await expect(textarea).toBeVisible({ timeout: 10000 })
+
+      // Should have the correct placeholder
+      await expect(textarea).toHaveAttribute(
+        'placeholder',
+        "What's on your mind?",
```

Comment:
Is it standard practice to hard code these? should we maybe make a constants file where everything that is tested goes? this way placeholders and checking the header (e.g., "Tinfoil Private Chat") verification is more robust to changes?